### PR TITLE
Updated to use centralised CI and standard version format for changelog

### PR DESCRIPTION
**Pull request by alourie**
Signed-off-by: Alex Lourie <alex.lourie@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [OPS-158 Standardise changelog format across all repos](https://app.clickup.com/t/6929263/OPS-158)

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### v0.4.0
+## [0.4.0]
```
Comment (kgreav) [inline changelog.md:1]: Does this need the  `- UNRELEASED`?

**Review by kgreav — CHANGES_REQUESTED** [review]
Can you please add the hotfix/LTS branch workflows

File: changelog.md (hunk 1 above)
Comment (alourie) [reply changelog.md]: yea, :facepalm:

**Review by kgreav — APPROVED** [review]

**Pull request merged**